types/verovio/verovio-tests.ts

```diff
@@ -0,0 +1,30 @@
+import * as verovio from 'verovio';
+import { AnyOption, IntOption } from 'verovio/AvailableOptions';
```

Comment:
```suggestion
import { AnyOption, IntOption } from 'verovio/AvailableOptions';
```
I cannot reach the public docs, but is there really a `/AvailableOptions` entity there on the Verovio module? Should it be just exported type on the package level? `import { AnyOption, IntOption } from 'verovio'`?

Reply:
Yes, you're right, this would make more sense.  I will update it